clients/fossid-webapp/src/main/kotlin/model/status/ScanDescription2021dot2.kt

```diff
@@ -23,7 +23,7 @@ import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import com.fasterxml.jackson.annotation.JsonProperty
 
 /**
- * A description of scan status. This class is for FossID version 2021.2.
+ * A description of scan status. This class is for FossID version 2021.2 and later.
```

Comment:
Nit: I guess the commit message type should be `docs`. Otherwise it's a bit unclear what aspect of the `ScanDescription2021dot2` class the "improved accuracy" refers to.